Library/Formula/byobu.rb

```diff
@@ -19,6 +20,9 @@ class Byobu < Formula
   depends_on "tmux"
   depends_on "newt" => "with-python"
 
+  depends_on "automake" => :build
```

Comment:
I think that we should use `if build.head?` when building head.
It seems that `automake` and `autoconf` don't have to install stable version.